pandaharvester/harvestercore/queue_config_mapper.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, queue_name):
         self.useJobLateBinding = False
         self.zipPerMB = None
         self.siteName = ""
+        self.site = ""
```

Comment:
What are the use cases when site and siteName are different?

Reply:
In harvester, I think the sitename = queuename.split("/")[0] (here queuename is pandaqueue/[SCORE/MCORE/UCORE]). It's already used. It's something like queuename.
For the site I added, I would like it to be something like SLAC, it has multiple queues SLAC_Rubin, SLAC_Rubin_Himem.

Reply:
I update the site to siteFamily.